tests/test_parity_blenderbot.py

```diff
@@ -0,0 +1,353 @@
+import unittest
```

Comment:
Don't think it's a good idea to add a dependency of `parlai` to `transformers`. It's great to have these tests during development of the model, but once the model works we should switch to hardcoded tests as they can be found for Bart, Reformer or Roberta to verify that the model works. 
IMO, there should be 2 kinds of hard-coded tests:

1) Hard coded tests for the forward function: INPUT: `input_ids` OUTPUT: `hidden_states` using `forward` -> test against some slice 
2) Hard coded generation tests: INPUT: `input string` OUTPUT: `output string` using `generate` -> test for equality.

Reply:
And I would prefer to put all the test code into `test_modeling_bart`, don't really see why we need to make an exception for Blenderbot to have two test files -> this is not intuitive given how the library is written at the moment.

Reply:
yes this was for development purposes only.